src/graphics/context/webgl.rs

```diff
@@ -60,7 +60,7 @@ impl Texture {
         gl_ctx: &WebGLRenderingContext,
         width: u16,
         height: u16,
-        bytes: &[u8],
+        bytes: Option<&[u8]>,
```

Comment:
I'm creating a `Texture` for my canvas with fixed width and height using a `None` here, which will be a null pointer in JS (which is intended). This appears to work fine as I'm able to render the texture and it just draws as transparent until I try to do anything with it.